MTA-2239 _ MTA 7.0.3 Release Notes (#869)
windup/windup-documentation: docs/topics/mta-rn-resolved-issues-7-0-3.adoc

// Module included in the following assemblies:
//
// * docs/release_notes-7.0/master.adoc

:_content-type: REFERENCE
[id="mta-rn-resolved-issues-7-0-3_{context}"]
= Resolved issues

> The following highlighted issues have been resolved in {ProductFullName} version 7.0.3.

Reviewer: ```suggestion
This section provides issues that have been resolved in {ProductFullName} version 7.0.3.
```

.Returned `Multi select filter` option in `All issues` page

The option has been returned to select several names from a drop-down list of applications. link:https://issues.redhat.com/browse/MTA-2560[(MTA-2560)]

.Fixed failed analysis on DC clusters

> Failed upload and analysis on a disconnected cluster has been fixed. The failed analysis returned the `Error: exit status 128` error. This issue has been resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2513[(MTA-2513)]

Reviewer: Shall we exclude the last sentence? It's in the section for 7.0.3, therefore, I think there's no need to repeat this info here.

```suggestion
Failed upload and analysis on a disconnected cluster has been fixed. The failed analysis returned the `Error: exit status 128` error. link:https://issues.redhat.com/browse/MTA-2513[(MTA-2513)]
```

.Fixed incorrect assessment status when running an assessment on two questionnaires 

> In {ProductShortName} 7.0.2, running two questionnaires showed the `Assessment` status as `Not started` instead of `In progress`. This has been fixed in {ProductShortName} 7.0.3, and now the Assessment status shows `In progress` after one questionnaire or Archetype is started. list:https://issues.redhat.com/browse/MTA-2409[(MTA-2409)] list:[(MTA-2410,] list:https://issues.redhat.com/browse/MTA-2410[MTA-2410)]

Reviewer: ```suggestion
In {ProductShortName} 7.0.2, running two questionnaires displayed the `Assessment` status as `Not started` instead of `In progress`. With this update, the problem has been fixed. As a result, the Assessment status shows `In progress` after one questionnaire or archetype is started. list:https://issues.redhat.com/browse/MTA-2409[(MTA-2409)] list:[(MTA-2410,] list:https://issues.redhat.com/browse/MTA-2410[MTA-2410)]
```

.Analysis fails on MacOS
// Copied from Known issues.

Running the `analyze` command on a MacOS machine does not work when Podman does not have enough memory. The analysis failed with timeout, without returning an error. 

> Workaround: Set Podman memory to at least 4GB. Podman on MacOS sets default memory to 2GB, which is not enough to run {ProductShortName}. link:https://issues.redhat.com/browse/MTA-2518[(MTA-2518)]

Reviewer: Should it be rephrased as a Resolved issue? Now it's still a Known Issue.

> .Tests failed when the `output.yaml` file was saved to a different directory than the default
> // Copied from Known issues.
> 
> When running a CLI command to analyze an application, the analysis required saving the `output.yaml` file only in the default directory. Setting the analysis to save to a different directory caused the analysis to fail and return an error. This issue was resolved in {ProductShortName} 7.0.3. list:https://issues.redhat.com/browse/MTA-2654[(MTA-2654)] 

Reviewer: Should it be rephrased as a Resolved issue? Now it's still a Known Issue.
```suggestion
.Tests do not fail when the `output.yaml` file is saved to a different directory than the default
// Copied from Known issues.

When running a CLI command to analyze an application, the analysis requires saving the `output.yaml` file only in the default directory. Previously, when you set the analysis to save `output.yaml` to a different directory, the analysis failed with an error. With this update, the issue has been fixed. As a result, tests do not fail when the `output.yaml` file is saved to a different directory than the default. list:https://issues.redhat.com/browse/MTA-2654[(MTA-2654)] 
```
